go.mod

```diff
@@ -5,6 +5,7 @@ go 1.17
 require (
 	github.com/PaesslerAG/jsonpath v0.1.1
 	github.com/cloudevents/sdk-go/v2 v2.5.0
+	github.com/gobwas/glob v0.2.3
```

Comment:
This package is super outdated, I wonder if there is a newer "better" package we can use.
@pchila any recommendations?